.travis.yml

```diff
@@ -61,6 +61,7 @@ git:
 before_script:
   - export PLUGIN_SLUG=$(basename $(pwd))
   - export PATH="$HOME/.composer/vendor/bin:$PATH"
+  - composer install
```

Comment:
Question: Should we have this here or https://github.com/Automattic/jetpack/blob/master/tests/setup-travis.sh#L16 ?